notable.go

```diff
@@ -2,20 +2,19 @@ package notable
 
 import (
 	"fmt"
-	slack "github.com/harvesthq/notable/Godeps/_workspace/src/github.com/nlopes/slack"
+	slack "github.com/nlopes/slack"
```

Comment:
Yup. I believe that Heroku used to require that but now [they support it](https://devcenter.heroku.com/articles/go-dependencies-via-godep)